html/pfappserver/root/static.alt/src/views/Nodes/_components/NodeView.vue

```diff
@@ -46,6 +49,9 @@
         </b-tab>
 
         <b-tab title="Fingerbank">
+          <template slot="title">
+            {{ $t('Fingerbank') }}
+          </template>
```

Comment:
Fingerbank will (hopefully) never be translated 😉 Remove the slot and simply keep the title attribute.